components/script/dom/htmlimageelement.rs

```diff
@@ -1542,6 +1557,31 @@ impl HTMLImageElementMethods for HTMLImageElement {
         }
     }
 
+    // https://html.spec.whatwg.org/multipage/#attr-img-referrerpolicy
```

Comment:
This link should be https://html.spec.whatwg.org/multipage/#dom-img-referrerpolicy.